src/dashboards/teachers/common.libsonnet

```diff
@@ -0,0 +1,115 @@
+// Video commons
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local template = grafana.template;
+local common = import '../common.libsonnet';
+
+{
+  fields: {
+    context_extensions_completion_threshold: 'context.extensions.https://w3id.org/xapi/video/extensions/completion-threshold',
+    result_extensions_length: 'result.extensions.https://w3id.org/xapi/video/extensions/length',
+    result_extensions_time: 'result.extensions.https://w3id.org/xapi/video/extensions/time',
+    verb_display_en_us: 'verb.display.en-US.keyword',
+  },
+  queries: {
+    course_key: 'context.contextActivities.parent.id.keyword:${EDX_COURSE_KEY:doublequote}',
+    templating_course_query: '%(course_key)s OR (%(school_course_session)s)' % {
+      course_key: $.queries.course_key,
+      school_course_session: $.queries.templating_school_course_session,
+    },
+    course_query: '%(course_key)s OR (%(school_course_session)s)' % {
+      course_key: $.queries.course_key,
+      school_course_session: $.queries.school_course_session,
+    },
+    edx_course_key: 'SELECT `key` FROM courses_course WHERE `key`="${EDX_COURSE_KEY}"',
+    templating_school_course_session: '%(school)s:${SCHOOL:doublequote} AND %(course)s:${COURSE:doublequote} AND %(session)s:${SESSION:doublequote}' % {
+      course: common.utils.double_escape_string(common.fields.course),
+      school: common.utils.double_escape_string(common.fields.school),
+      session: common.utils.double_escape_string(common.fields.session),
+    },
```

Comment:
Why not using `std.strReplace(my_var, '\', '\\')` where required to avoid duplicated, hard to maintain code?